bootstrap.sh

```diff
@@ -29,13 +29,13 @@ fi
 
 ################ parameters ################
 # directory where the script is located
-SCRIPT_DIR=$( cd $(dirname $0); pwd -P)
+SCRIPT_DIR="$( cd $(dirname $0); pwd -P)"
```

Comment:
The quotes have no effect here. They are parsed by the script interpreter and will not wrap the resulting path in quotes. Try with:

```bash
SCRIPT_DIR="\"$( cd $(dirname $0); pwd -P)\""
```